gotomagic/githubmagic.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+import requests
```

Comment:
When doing a fresh install of this branch i got a `missing requests error`
```
jonas in ~/git/technocake/goto on feature/github-autocompletions
$ sudo ./uninstall.sh && sudo ./install.sh 
remove the line: source start_goto from your bash config file
(most likely .bash_profile or .bashrc in your home folder)

 Also, your data in ~/.goto is not deleted
Step 1: Installing goto into /usr/local/opt/goto
Step 2: Adding symlinks to /usr/local/bin
Step 3: Setting up magic data folder in /home/jonas/.goto
~/.bash_profile does not exist

To make goto function properly, add this line to your bash config file: 

         source start_goto

into one of these (.bashrc | .profile | .bash_profile)
Want to append to .bashrc? [y|n]: y
active project is now: goto
active project is now: goto
Traceback (most recent call last):
  File "/usr/local/opt/goto/the_real_goto.py", line 9, in <module>
    from gotomagic.githubmagic import GitHub
  File "/usr/local/opt/goto/gotomagic/githubmagic.py", line 2, in <module>
    import requests
ImportError: No module named requests
ERROR: Installation step failed with exit code 1
```

Reply:
UPDATE
I ran pip install -r requirements.txt and got the same result... wierd

Reply:
**UPDATE** - Today pip2 works! Pip3 still not working
**pip3**: https://asciinema.org/a/y4R65wKjtsDxY1OvmFIznZYdi
**pip2**: https://asciinema.org/a/qLCM0R42RyKdTOqtiu3TmiZbV

Reply:
My python versions
```
$ python --version
Python 2.7.15rc1

jonas in ~/git/technocake/goto on feature/github-autocompletions
$ python3 --version
Python 3.6.5
```

Reply:
goto uses "python" from the environment, not python3 so I guess you install requests with pip3 but goto finds python, which is symlinked to python2.7 -- and requests isn't installed there

Reply:
Until this point, I made effort into making goto dependency-less - to solve these issues. 
But it limits its design. I could implement standard python versions of both  fetching urls (using urllib) and copying to clipboard ( using tkinter) -- it would remove dependencies. 

Open to discuss. 

Think structuring goto as a python package is the way to go if we choose to use dependencies

Reply:
Interesting discussion indeed. The idea of having as few dependencies as possible is very attractive. If Python has standard libraries which solves our problem (without having to jump through hoops) why not use them?

If standard libraries does not solve our problem, we have to look somewhere else.